docs/setup.asciidoc

```diff
@@ -1,33 +1,56 @@
 [[setup]]
-== Getting Kibana Up and Running
+== Installing Kibana
 You can set up Kibana and start exploring your Elasticsearch indices in minutes.
 All you need is:
 
 * Elasticsearch {esversion}
-* A modern web browser - http://www.elastic.co/subscriptions/matrix#matrix_browsers[Supported Browsers].
+* A modern web browser - https://www.elastic.co/support/matrix#show_browsers[Supported Browsers].
 * Information about your Elasticsearch installation:
 ** URL of the Elasticsearch instance you want to connect to.
 ** Which Elasticsearch indices you want to search.
 
 NOTE: If your Elasticsearch installation is protected by http://www.elastic.co/overview/shield/[{scyld}], see
 {shield}/kibana.html#using-kibana4-with-shield[{scyld} with Kibana] for additional setup instructions.
 
+=== Upgrading Kibana
+
+Your existing Kibana version is generally compatible with the next minor version release of Elasticsearch.
+This means you should upgrade your Elasticsearch cluster(s) before or at the same time as Kibana.
+We cannot guarantee compatibility between major version releases so in those cases both Elasticsearch and Kibana must be upgraded together.
+
+. https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html[snapshot] existing `.kibana` index in Elasticsearch
+. save a backup copy of the existing `config/kibana.yml`
+. take note of the Kibana plugins that are installed;
```

Comment:
@debadair I'm not clear on the bullets comment.  All of the numbered lists should be changed to bullets?  Several sections had numbered lists before.  Are you saying I should change them all or just the Upgrading sections?